src/webserver/api/get_calendar_week.php

```diff
@@ -1,39 +1,56 @@
 <?php
-    $format = "d.m.Y";
+    //input: http://localhost/api/get_calendar_week.php?calendarWeek=17&year=2022
+    //output: {"calendarWeek":"51","weekdays":["20.12.2021","21.12.2021","22.12.2021","23.12.2021","24.12.2021"],"year":"2021"}
+    
+    //check if file is called directly
+    if (basename(__FILE__) == basename($_SERVER["SCRIPT_FILENAME"])) {
+        //calculate offset based on calendarWeek
+        if(!isset($_GET["calendarWeek"]) || empty($_GET["calendarWeek"])) {
```

Comment:
Probier mal das hier bitte
```suggestion
        if(empty($_GET["calendarWeek"])) {
```